Cargo.toml

```diff
@@ -90,9 +94,10 @@ terminal_size = "0.3.0"
 timeago = { version = "0.4.2", default-features = false }
 unicode-width = "0.1"
 zoneinfo_compiled = "0.5.1"
-rayon = "1.10.0"
 ansi-width = "0.1.0"
-
+serde = { version = "1.0.193", features = ["derive"] }
+serde_yaml = "0.9.29"
```

Comment:
This was actually deprecated, I have a fork, but let's not switch to that rn, we can do that in another PR if we want (https://crates.io/crates/serde_norway)